src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const propertiesArray = sourceString.split(';');
+
+  return propertiesArray.reduce((style, current) => {
+    const [key, value] = current.split(':');
+
+    if (!!key.trim() || value) {
+      style[key.trim()] = value.trim();
+    }
```

Comment:
Try to put repetitive parts of the code in variables. It's also worth checking both the `key` and the `value`.
```suggestion 
    const property = key.trim();
    const propertyValue = value.trim();
    if (property && propertyValue) {
      style[property] = propertyValue;
    }
```